Files src5.1.3.orig/YACS_SRC_5.1.3/adm/unix/config_files/.ac_python_devel.m4.swp and src5.1.3/YACS_SRC_5.1.3/adm/unix/config_files/.ac_python_devel.m4.swp differ
diff -ur src5.1.3.orig/YACS_SRC_5.1.3/adm/unix/config_files/ac_python_devel.m4 src5.1.3/YACS_SRC_5.1.3/adm/unix/config_files/ac_python_devel.m4
--- src5.1.3.orig/YACS_SRC_5.1.3/adm/unix/config_files/ac_python_devel.m4	2010-03-08 20:51:25.810829226 +0100
+++ src5.1.3/YACS_SRC_5.1.3/adm/unix/config_files/ac_python_devel.m4	2010-03-08 20:53:41.330413913 +0100
@@ -48,7 +48,7 @@
 	# Check for Python library path
 	AC_MSG_CHECKING([for Python library path])
 	python_path=`echo $PYTHON | sed "s,/bin.*$,,"`
-	for i in "$python_path/lib/python$PYTHON_VERSION/config/" "$python_path/lib/python$PYTHON_VERSION/" "$python_path/lib/python/config/" "$python_path/lib/python/" "$python_path/" ; do
+	for i in "$python_path/lib$LIB_LOCATION_SUFFIX" "$python_path/lib$LIB_LOCATION_SUFFIX/python$PYTHON_VERSION/config/" "$python_path/lib$LIB_LOCATION_SUFFIX/python$PYTHON_VERSION/" "$python_path/lib$LIB_LOCATION_SUFFIX/python/config/" "$python_path/lib$LIB_LOCATION_SUFFIX/python/" "$python_path/" ; do
 		python_path=`find $i -type f -name libpython$PYTHON_VERSION.* -print | sed "1q"`
 		if test -n "$python_path" ; then
 			break
